src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$clock-height: 80vmin;
+$clock-width:80vmin;
+$hand-width: 2vmin;
+$border-color: 1px solid rgb(0, 255, 17);
```

Comment:
Be more descriptive when creating variables, if you are creating a color variable describe the color (light-grey, black, light-blue etc.) also this here is not a color alone but rather a whole border.